torch/nn/parallel/distributed.py

```diff
@@ -859,7 +859,7 @@ def __init__(
             _setup_mixed_precision_params(self.mixed_precision, self.module)
             _cast_buffers(self.mixed_precision, self.module)
             # Stream used for async low precision copies.
-            self._mp_stream = torch.cuda.Stream()
+            self._mp_stream = torch.Stream()
```

Comment:
for my education, what does creating a `torch.Stream()` without specifying a device actually do?  Does it rely on some notion of 'current device' to be active? or default to 'cuda'?

Reply:
@wconstab If no device information is provided, `torch.Stream` will 'create' a stream on the current device associated with the active accelerator. Refer to https://github.com/pytorch/pytorch/blob/9e0b3e9b6c1e4ddaeabad10b5fff4c8bf7d22be9/torch/csrc/Stream.cpp#L53-L56